.github/workflows/pr-builds.yml

```
@@ -25,77 +25,69 @@ jobs:
       - name: Build cross-platform binaries
         run: ./scripts/build.sh
 
-      - name: Upload Linux AMD64
-        uses: actions/upload-artifact@v4
+      - name: Create or update pre-release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)